<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PNFEM Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>15</y>
      <width>978</width>
      <height>508</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="hlabel">
       <property name="text">
        <string>h [m]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="hEdit"/>
     </item>
     <item row="0" column="7" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="showGeometryButton">
       <property name="text">
        <string>Geometry</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="aLabel">
       <property name="text">
        <string>a [m]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="aEdit"/>
     </item>
     <item row="1" column="9">
      <widget class="QPushButton" name="showMeshButton">
       <property name="text">
        <string>Mesh</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="bLabel">
       <property name="text">
        <string>b [m]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="bEdit"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="bendLabel">
       <property name="text">
        <string>b, end</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="bendEdit"/>
     </item>
     <item row="2" column="5">
      <widget class="QRadioButton" name="bendButton">
       <property name="text">
        <string>Vary</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="7" colspan="2">
      <widget class="QCheckBox" name="undisplacedBox">
       <property name="text">
        <string>Show undisplaced mesh</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QPushButton" name="showDisplacementsButton">
       <property name="text">
        <string>Displacements</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="wLabel">
       <property name="text">
        <string>w [m]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="wEdit"/>
     </item>
     <item row="3" column="9">
      <widget class="QPushButton" name="showElementValuesButton">
       <property name="text">
        <string>Element effective stresses</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="tlabel">
       <property name="text">
        <string>t [m]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="tEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="eLabel">
       <property name="text">
        <string>E [Pa]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="eEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="vLabel">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="vEdit"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="qLabel">
       <property name="text">
        <string>q [N]</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="qEdit"/>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="qendLabel">
       <property name="text">
        <string>q, end</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLineEdit" name="qendEdit"/>
     </item>
     <item row="7" column="5">
      <widget class="QRadioButton" name="qendButton">
       <property name="text">
        <string>Vary</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="paramstudyLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Param steps</string>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="QSpinBox" name="paramstudyBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="7" column="9">
      <widget class="QPushButton" name="paramstudyButton">
       <property name="text">
        <string>Param study</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="maxElSizeLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Max el. area</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QSlider" name="maxElSizeSlide">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="sliderPosition">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QLineEdit" name="maxElSizeEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QLabel" name="elTypeLabel">
       <property name="text">
        <string>El. type</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="10">
      <widget class="QPlainTextEdit" name="reportEdit"/>
     </item>
     <item row="8" column="7">
      <widget class="QRadioButton" name="triButton">
       <property name="text">
        <string>Triangle</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="8" column="8">
      <widget class="QRadioButton" name="quadButton">
       <property name="text">
        <string>Quad</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="8" column="9">
      <widget class="QPushButton" name="executePushButton">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCalc">
    <property name="title">
     <string>Calc</string>
    </property>
    <addaction name="actionExecute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
